packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -353,6 +355,16 @@ class EditableText extends StatefulWidget {
   /// {@endtemplate}
   final bool obscureText;
 
+  /// {@template flutter.widgets.editableText.readOnly}
+  /// Whether the text can be changed.
+  ///
+  /// When this is set to true, the text cannot be modified
+  /// by any shortcut or keyboard operation. The text is still selectable.
+  ///
+  /// Defaults to false. Cannot be null.
```

Comment:
I believe "Cannot be null" should be "Must not be null" - but please double check with a native english speaker.

Reply:
I copied from other bool properties in the class, do we want to update all of them?

Reply:
Yes.  `Must not` is better.  `Cannot` implies it's impossible for these things to happen - `Must not` implies that it's possible but you are never to do it (and doing so will cause errors).

I would separate updating the rest of the language out into a separate PR though.

Reply:
fixed